vmhost/contexts/output.go

```diff
@@ -401,12 +397,17 @@ func (context *outputContext) Transfer(
 		context.WriteLogWithIdentifier(
 			sender,
 			[][]byte{value.Bytes(), destination},
-			[][]byte{[]byte(""), input},
+			[][]byte{[]byte("DirectCall"), input},
```

Comment:
create a const for this and use it also on L435?

Reply:
Done